component/tests/golden/apiserver/appcat/appcat/10_appcat_backup_monitoring.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: appcat-backup
+  namespace: syn-appcat
+spec:
+  groups:
+    - name: appcat-backup
+      rules:
+        - alert: AppCatBackupJobError
+          annotations:
+            description: The backup job {{ $labels.job_name }} in namespace {{ $labels.namespace
+              }} has failed.
+            runbook_url: https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatBackupJobError.html
+            summary: AppCat service backup failed.
+          expr: kube_job_failed{job_name=~".*backup.*", namespace=~"vshn-(postgres|redis|stsResizer)-.*"}
```

Comment:
Shouldn't it be postgresql instead of posgres? Why there's stsResizer as well?

Reply:
As discussed in chat, I will refactor the `services.vshn` object, so it only contains actual vshn services.